docs/platform/server-v3/modules/about.md

```diff
@@ -14,100 +14,132 @@ enterprise_only: true
+- `@deity/falcon-X-module` that provides an implementation of all the required things in one package
 
 ## Module and extension relation
 
-Please keep in mind that extension packages are still needed, as extension provide GraphQL schema that will be exposed to GraphQL clients. 
-So now, the 2 things need to be provided:
+Please keep in mind that extension packages are still needed, as extensions provide GraphQL schema that will be exposed to GraphQL clients.
+So now, the two things need to be provided:
+
 - extension which is an "interface" for a particular feature
 - module which is an "implementation" of that feature
 
-## Custom modules
+## Configuring a module
+
+In order to enable a module in Falcon Server, you need to add its configuration into the `config.json` (normally `server/config/default.json`) file under the `"modules"` section.
+
+```json
+{
+  "modules": {
+    "my-module": {
+      "package": "<path>"
+    }
+  }
+}
+```
 
-All the integrations available in Falcon Platform are implemented as extensions + modules. When you want to add new features or change the existing behavior you'll need to add [Extension](../extensions/about) and module that implements features for that Extension.
+As you can see, Falcon Server expects a key-value map of modules in most cases, the order does not matter.
 
-Modules can be registered in 2 ways - with auto-discovery mechanism or manually.
+We recommend adding your base configuration in `server/config/default.json` file so it will be available no matter what mode (`development`, `production` or any custom) Falcon Server is running in. If a module requires some credentials that should be secret and not committed to your git repository then that configuration can be added in `server/config/local.json` for local development and as environment variables for remote apps. See the configuration guide for more details.
 
-### Adding module
+Module configuration can be described in following way:
 
-In order to enable a module in Falcon Server you need to first add it in the configuration file under `"modules"` section. We recommend adding it in `config/default.json` file so it will be available no matter what mode (development, production or custom) Falcon Server is running. 
-If a module requires some credentials or secrets (sensitive configuration) then that configuration can be added in `config/local.json`. See configuration guide for more details.
+```ts
+type ModuleConfiguration = {
+  package: string;
+  enabled?: boolean;
+  config?: Record<string, any>;
+};
+```
+
+- `package: string` - path to Module entry point, tells Falcon Server wherefrom the module should be loaded. It can be an `npm` package name (e.g. `@deity/falcon-magento2-module`), which of course needs to be installed, so added to `package.json` file it can be `npm` package name or path relative to the root of your server application to a local module placed inside application folder (e.g. `./src/my-module`).
```

Comment:
```suggestion
- `package: string` - path to Module entry point, tells Falcon Server wherefrom the module should be loaded. It can be an `npm` package name (e.g. `@deity/falcon-magento2-module`), which of course needs to be installed, so added to `package.json` file or path relative to the root of your server application to a local module placed inside application folder (e.g. `./src/my-module`).
```

Reply:
done